test/mutations/test_asset.py

```diff
@@ -105,11 +113,18 @@ def setUp(self):
         # Create a temporary directory
         self.test_dir = tempfile.mkdtemp()
 
-    def tearDown(self):
+    def tearDown(self, mocker):
         # Remove the directory after the test
         shutil.rmtree(self.test_dir)
 
-    def test_geotiff_upload_properties(self):
+        mocker.patch(
```

Comment:
why did the method `test_geotiff_upload_properties` go away? `tearDown` is used to clean-up the test environment.

Reply:
that's a mistake, I put it back